### ISSUE TITLE
Form 26 not showing up in Names DB

### ISSUE THREAD

**Issue by sarabeckman** [densho/namesdb#6]
Form 26 data isn't showing up in results when the Names DB is searched. Form 26 data also isn't showing on pages that lists family members when looking at a FAR record. 

See:  http://ddr.densho.org/names/far-minidoka:8-minidoka_hoshide_1917_hideo/

**Comment by GeoffFroh** [comment]
Looking at the ES7 cluster, it looks like the `wra-master` dataset was not indexed in. There are only 191021 docs in the index; where on the old ES2 cluster, there are more than 301K. 

Also can't get any docs back querying for "wra" against the `_doc.m_dataset` field. The same query on the ES2 cluster returns 151K+ hits.

**Comment by gjost** [comment]
I'm sorry I don't know what the Form 26 data should look like.  What should I be looking for?

UPDATE: It's the data in `namesdb-data/wra-master.csv`.  Derp.

**Comment by gjost** [comment]
Looks like a problem parsing the `namesdb-data/wra-master.csv` file:
```
(ddrpublic) ddr@lightning:/opt/ddr-public$ namesdb post -H 192.168.0.21:9200 namesdb-data/wra-master.csv                                                                     
2020-01-23 11:46:25,617 INFO     Dataset: wra-master                                                                                                                         
2020-01-23 11:46:25,617 INFO     Fields: ['m_dataset', 'm_pseudoid', 'm_camp', 'm_lastname', 'm_firstname', 'm_birthyear', 'm_gender', 'm_originalstate', 'm_familyno', 'm_ind
ividualno', 'm_altfamilyid', 'm_altindividualid', 'm_ddrreference', 'm_notes', 'w_assemblycenter', 'w_originaladdress', 'w_birthcountry', 'w_fatheroccupus', 'w_fatheroccupabr
', 'w_yearsschooljapan', 'w_gradejapan', 'w_schooldegree', 'w_yearofusarrival', 'w_timeinjapan', 'w_notimesinjapan', 'w_ageinjapan', 'w_militaryservice', 'w_maritalstatus', '
w_ethnicity', 'w_birthplace', 'w_citizenshipstatus', 'w_highestgrade', 'w_language', 'w_religion', 'w_occupqual1', 'w_occupqual2', 'w_occupqual3', 'w_occuppotn1', 'w_occuppot
n2', 'w_filenumber']                                                                                                                                                         
2020-01-23 11:46:25,617 INFO     Reading file: namesdb-data/wra-master.csv                                                                                                   
2020-01-23 11:46:31,232 INFO     ok (109212 rows)                                                                                                                            
2020-01-23 11:46:31,233 INFO     Verifying headers                                                                                                                           
2020-01-23 11:46:31,233 INFO     ok                                                                                                                                          
2020-01-23 11:46:31,233 INFO     Loading records                                                                                                                             
2020-01-23 11:46:31,234 INFO     Loading 1/109212 <Record wra-master:2-poston_aanagawa_1927_mary>                                                                            
2020-01-23 11:46:31,235 INFO     Loading 2/109212 <Record wra-master:3-gilariver_aawato_1889_hisakich>                                                                       
...
2020-01-23 11:42:52,423 INFO     Loading 53842/109212 <Record wra-master:6-jerome_nagamine_1919_yasanobu>                                                                    
Traceback (most recent call last):
  File "/opt/ddr-public/venv/ddrpublic/bin/namesdb", line 14, in <module>
    load_entry_point('namesdb==1.0.0', 'console_scripts', 'namesdb')()
  File "/opt/ddr-public/venv/ddrpublic/lib/python3.5/site-packages/click/core.py", line 764, in __call__                                                                     
    return self.main(*args, **kwargs)
  File "/opt/ddr-public/venv/ddrpublic/lib/python3.5/site-packages/click/core.py", line 717, in main                                                                         
    rv = self.invoke(ctx)
  File "/opt/ddr-public/venv/ddrpublic/lib/python3.5/site-packages/click/core.py", line 1137, in invoke                                                                      
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/opt/ddr-public/venv/ddrpublic/lib/python3.5/site-packages/click/core.py", line 956, in invoke                                                                       
    return ctx.invoke(self.callback, **ctx.params)
  File "/opt/ddr-public/venv/ddrpublic/lib/python3.5/site-packages/click/core.py", line 555, in invoke                                                                       
    return callback(*args, **kwargs)
  File "/opt/ddr-public/venv/ddrpublic/lib/python3.5/site-packages/namesdb-1.0.0-py3.5.egg/namesdb/cli.py", line 145, in post                                                
    publish.import_records(ds, dataset, stop, csvpath)
  File "/opt/ddr-public/venv/ddrpublic/lib/python3.5/site-packages/namesdb-1.0.0-py3.5.egg/namesdb/publish.py", line 155, in import_records                                  
    records = load_records(dataset, fields, headers, rows)
  File "/opt/ddr-public/venv/ddrpublic/lib/python3.5/site-packages/namesdb-1.0.0-py3.5.egg/namesdb/publish.py", line 97, in load_records                                     
    logging.info('Loading %s/%s %s' % (n, num_rows, record))
  File "/opt/ddr-public/venv/ddrpublic/lib/python3.5/site-packages/namesdb-1.0.0-py3.5.egg/namesdb/models.py", line 103, in __repr__                                         
    return "<Record %s>" % Record.make_id(self.m_dataset, self.m_pseudoid)
  File "/opt/ddr-public/venv/ddrpublic/lib/python3.5/site-packages/namesdb-1.0.0-py3.5.egg/namesdb/models.py", line 107, in make_id                                          
    return ':'.join([m_dataset, m_pseudoid])
TypeError: sequence item 1: expected str instance, NoneType found
```

**Comment by GeoffFroh** [comment]
Fixed bad data in wra-master and reindexed. All data for `namesdb` should come from the `namesdb-data` repo

**Comment by gjost** [comment]
Modified `namesdb post` to handle errors gracefully and print out defective CSV rows at the end of a run.  Fixed in `namesdb` commit `7b92c64` for next `ddrpublic` package.
